automatic/ruby.portable/legal/LICENSE.txt

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2007-2014, RubyInstaller Team
```

Comment:
This license looks a bit outdated, is this really the latest license available?

Reply:
Just the year is now 2020, no other changes. The URL in the verification was outdated, I've fixed that up, and will fix it up for the other ruby packages as well.